bigquery_etl/cli/backfill.py

```diff
@@ -355,40 +350,47 @@ def process(ctx, qualified_table_name, sql_dir, project_id, dry_run):
     if backfills_to_process_dict:
         entry_to_process = backfills_to_process_dict[qualified_table_name]
 
+        click.echo(f"\nValidating backfill for {qualified_table_name} via dry run:")
+        _process_backfill(ctx, qualified_table_name, entry_to_process, dry_run=True)
+
+        click.echo(f"\nProcessing backfills for {qualified_table_name}:")
+        _process_backfill(ctx, qualified_table_name, entry_to_process)
+
+        click.echo(f"Backfill processing completed for {qualified_table_name}.")
+
+
+def _process_backfill(ctx, qualified_table_name, entry_to_process, dry_run=None):
+    project, dataset, table = qualified_table_name_matching(qualified_table_name)
+
+    backfill_staging_qualified_table_name = None
```

Comment:
nit: why initialize to `None` here? I'm not seeing it ever used as `None`/empty? 
```suggestion
    backfill_staging_qualified_table_name = (
            get_backfill_staging_qualified_table_name(
                qualified_table_name, entry_to_process.entry_date
            )
        )
```